spdy_test.go

```diff
@@ -252,6 +252,91 @@ func TestHalfClose(t *testing.T) {
 	wg.Wait()
 }
 
+func TestUnexpectedRemoteConnectionClosed(t *testing.T) {
+	tt := []struct {
+		closeReceiver bool
+		closeSender   bool
+	}{
+		{closeReceiver: true, closeSender: false},
+		{closeReceiver: false, closeSender: true},
+		{closeReceiver: false, closeSender: false},
+	}
+	for tix, tc := range tt {
+		listen := "localhost:7743"
```

Comment:
Can you rotate the listen port here.  I am getting listen failures on my machine.

Something like this worked for me `listen := fmt.Sprintf("localhost:774%d", tix)`